packages/grafana-runtime/package.json

```diff
@@ -40,6 +40,7 @@
     "@grafana/data": "10.2.0-pre",
     "@grafana/e2e-selectors": "10.2.0-pre",
     "@grafana/faro-web-sdk": "1.1.2",
+    "@grafana/schema": "workspace:^",
```

Comment:
Can we use the version protocol (e.g. `10.2.0-pre`) here please? I'm scared the version of Lerna we use doesn't understand this and will go pop when a release is made.

Reply:
Thanks for noticing! Fixed.